frontend/simple/js/state.js

```diff
@@ -276,7 +299,7 @@ const actions = {
     }
```

Comment:
The contents of `loadSettings` should be moved into `login`, and `commit('login', user)` should be done at the very end of the function. Also double-check NavBar logic for displaying SignIn/SignOut button + related test case wait scenarios. We should be able to remove the reliance on the queue.